packages/office-ui-fabric-react/src/components/Icon/Icon.tsx

```diff
@@ -42,6 +42,23 @@ export const Icon = (props: IIconProps): JSX.Element => {
         <Image { ...props.imageProps as any } />
       </div>
     );
+  } else if (typeof iconName === 'string' && iconName.length === 0) {
```

Comment:
Is an entire new `else if` block needed for this? Can't you just toggle the placeholder classes?

Reply:
I think there'd be some markup/accessibility differences between an icon and something with no value other than holding the place for one. We can clean up in the future if not needed.